### PR TITLE
Update gemspec 

### PR DESCRIPTION
**Pull request by jamescway**
Move jruby dependencies into platform == java block in the gemspec.

#99

### CODE REVIEW

**Comment by rtyler** [comment]
didn't work on non-jruby

File: spec/integration/producer_spec.rb [hunk 1]
```diff
@@ -14,7 +17,7 @@
   let(:timeout) { 10 }
   let(:message)   { 'msg' }
   let(:consumer) do
-    Hermann::Consumer.new(topic, "rspec-group", zookeepers)
+    Hermann::Consumer.new(topic, {:group_id => "rspec-group", :zookeepers => zookeepers})
```
Comment (mkristian) [inline spec/integration/producer_spec.rb]: looks off topic to  this PR

File: spec/integration/producer_spec.rb [hunk 2]
```diff
@@ -3,7 +3,10 @@
 require 'hermann/producer'
 require 'hermann/consumer'
 require 'hermann/discovery/zookeeper'
-require 'concurrent'
+
+if Hermann.jruby?
+  require 'concurrent'
```
Comment (rtyler) [inline spec/integration/producer_spec.rb]: Shouldn't this move into `hermann/java`?

**Pull request closed without merging**